drift.c

```diff
@@ -11,19 +11,19 @@
 #include "endrun.h"
 
 
-static int drift_particle_full(int i, int ti1, int blocking);
+static int drift_particle_full(int i, unsigned int ti1, int blocking);
```

Comment:
Could do that indeed. I did not just because it gets a bit messy with header includes (and is a bit painful in terms of typing).